src/Model/Item.php

```diff
@@ -2228,6 +2228,9 @@ public static function addShadowPost($itemid)
 	 */
 	private static function addLanguageToItemArray(&$item)
 	{
+		// @TODO Find out why body can be empty here
```

Comment:
To find this out, an exception can be thrown when `$item['body']` is empty.

Reply:
Good idea, however I won't add an Exception in the develop branch. It should be used for debugging only.

Reply:
For debugging only, exceptions are never the choice. They are like hard errors, like a missing file but it was expected. Then `assert(isset($item['body']));` is here maybe the right choice. Still, if `body` is absent, an `E_NOTICE` is later triggered. Or use `array_key_exists()`.

Can you please add `array` as type-hint here? Also can you please add `@param` documentation?

Reply:
We should possibly just move ```self::addLanguageToItemArray($item);``` some lines to the bottom where the bad had already been checked for content.

P.S.: I suggest just logging the callstack. Then we can check for the origin. I strongly oppose to using an exception here that would prevent the item from being stored. This would mean data loss.

Reply:
Good idea to move the call.